### PR TITLE
Create GoNewCoupon.com.xml

### PR DESCRIPTION
**Pull request by MojtabaFaal**

### CODE REVIEW

File: src/chrome/content/rules/GoNewSoft.com.xml [hunk 1]
```diff
@@ -1,7 +0,0 @@
-<ruleset name="GoNewSoft.com">
-        <target host="gonewsoft.com" />
-        <target host="www.gonewsoft.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/GoNewSoft.com.xml]: Please don't remove these targets, they are still working.

Comment (MojtabaFaal) [inline src/chrome/content/rules/GoNewSoft.com.xml]: ok

Comment (MojtabaFaal) [inline src/chrome/content/rules/GoNewSoft.com.xml]: please add "GoNewCoupon.com.xml" to https-everywhere

Comment (MojtabaFaal) [inline src/chrome/content/rules/GoNewSoft.com.xml]: thanks

Comment (J0WI) [inline src/chrome/content/rules/GoNewSoft.com.xml]: Could you update this PR that `gonewsoft.com` will not be removed?

Comment (MojtabaFaal) [inline src/chrome/content/rules/GoNewSoft.com.xml]: please
do not remove GoNewSoft.com.xml (Kept)
and add "GoNewCoupon.com.xml"

Comment (J0WI) [inline src/chrome/content/rules/GoNewSoft.com.xml]: So then you need to revert 1b127c4, otherwise I can not merge your branch without removing it.

Comment (MojtabaFaal) [inline src/chrome/content/rules/GoNewSoft.com.xml]: Hello
I do not know what to do
Please mute your removal request
Or exactly tell me what to

Comment (J0WI) [inline src/chrome/content/rules/GoNewSoft.com.xml]: Edit [this](https://github.com/MojtabaFaal/https-everywhere/blob/master/src/chrome/content/rules/GoNewCoupon.com.xml) file and add `<target host="gonewsoft.com" />` and `<target host="gonewsoft.com" />`.

**Comment by jeremyn** [comment]
@MojtabaFaal Are you still interested in working on this pull request?

**Comment by MojtabaFaal** [comment]
hello
please delete domains:
GoNewCoupon.com
GoNewSoft.com
thanks

**Comment by MojtabaFaal** [comment]
hello
GoNewCoupon.com
GoNewSoft.com
This domain no longer works
Thank
________________________________

**Comment by jeremyn** [comment]
@MojtabaFaal I think you are saying that you want to close this pull request because the domains https://gonewcoupon.com and https://gonewsoft.com don't work. I can confirm that https://gonewcoupon.com gets an HTTP `526` and https://gonewsoft.com times out. So, I'm closing this pull request and also your related pull request https://github.com/EFForg/https-everywhere/pull/13944.

**Pull request closed without merging**